doc/edge-metrics-v2.md

```diff
@@ -0,0 +1,43 @@
+# IoT Edge Metrics
+
+## IoT Edge Hub
+
+The IoT Edge Hub module exposes a number of metrics in the Prometheus exposition format that provide insights into its operational state.
+
+### How to enable
+
+As of release 1.0.9, metrics are exposed as an experimental feature available at http port **9600** of the Edge Hub module. To enable, the following environment variables should be set for the module (make note of the double underscores):
+
+| Environment Variable Name                | value  |
+|------------------------------------------|--------|
+| `ExperimentalFeatures__Enabled`          | `true` |
+| `ExperimentalFeatures__EnableMetrics`    | `true` |
+
+### Metrics
+
+| Name                                                        | Dimensions                                                                                                                                                                           | Description                                                                                                                                                                                              | Type    |
```

Comment:
Is it just me or is this getting formatted a bit weirdly by github?

Reply:
I used [Markdown Table Generator](https://www.tablesgenerator.com/markdown_tables) maybe that is what it is.